src/components/KFileUpload/KFileUpload.vue

```diff
@@ -156,19 +157,19 @@ const props = defineProps({
     default: null,
   },
   /**
-     * Set icon size
-     */
+    * Set icon size
+    */
   iconSize: {
     type: String,
-    default: '26',
+    default: KUI_ICON_SIZE_50,
```

Comment:
issue: I don't believe you can directly reference an imported const in prop declarations.

If I missed this in other PRs, we'll need to fix there as well.

Reply:
Why not? It seems to be working fine (maybe need to pull it into consuming app to confirm, is that what you're thinking?)

Also yes, I will fix it the other places

Reply:
Hmm... I saw an error when I did this in the `icons` repository. Let me try again over there and see if I'm misremembering something